### PR TITLE
Added some code formatter in pre-requisites

### PR DESCRIPTION
**Pull request by am-Anmol**
Targets #927 

### Description

Added requested code formatted with links in pre-requisites

### Checklist:

- [x] I have performed a self-review of my own code.

### CODE REVIEW

File: docs/prerequisites.md [hunk 1]
```diff
@@ -54,3 +54,19 @@ You can install it with the following ways:
...
 
 - [Laravel Module Utility Commands](https://nwidart.com/laravel-modules/v1/advanced-tools/artisan-commands)
+
+### 8. Code Formatter :butterfly:
+
+Add some of code formatter for automatic code styling:
+
+- Visual Studio Code Editor:
+  - [Prettier - for JS,CSS,HTML files](https://prettier.io/)
+  - [Laravel Blade Snippets for Balde files](https://marketplace.visualstudio.com/items?itemName=onecentlin.laravel-blade)
```
Comment (GauravGusain98) [inline docs/prerequisites.md:64]:
```suggestion
  - [Laravel Blade Snippets for Blade files](https://marketplace.visualstudio.com/items?itemName=onecentlin.laravel-blade)
```

File: docs/prerequisites.md [hunk 2]
```diff
@@ -54,3 +54,19 @@ You can install it with the following ways:
 This project uses Laravel Modules, so a basic understanding of the commands is recommended:
 
 - [Laravel Module Utility Commands](https://nwidart.com/laravel-modules/v1/advanced-tools/artisan-commands)
+
+### 8. Code Formatter :butterfly:
+
+Add some of code formatter for automatic code styling:
```
Comment (GauravGusain98) [inline docs/prerequisites.md:60]:
```suggestion
Add the plugins mentioned below for automatic code styling:
```

File: docs/prerequisites.md [hunk 3]
```diff
@@ -54,3 +54,19 @@ You can install it with the following ways:
...
+
+- Visual Studio Code Editor:
+  - [Prettier - for JS,CSS,HTML files](https://prettier.io/)
```
Comment (GauravGusain98) [inline docs/prerequisites.md:63]:
```suggestion
  - [Prettier - for JS, CSS, HTML files](https://prettier.io/)
```

File: docs/prerequisites.md [hunk 4]
```diff
@@ -54,3 +54,19 @@ You can install it with the following ways:
...
+  - [Laravel Blade Snippets for Balde files](https://marketplace.visualstudio.com/items?itemName=onecentlin.laravel-blade)
+  - [Laravel Extra Intellisense](https://marketplace.visualstudio.com/items?itemName=amiralizadeh9480.laravel-extra-intellisense)
+  - [PHP Intelephense](https://marketplace.visualstudio.com/items?itemName=bmewburn.vscode-intelephense-client)
+  - [PHP cs fixer](https://marketplace.visualstudio.com/items?itemName=junstyle.php-cs-fixer)
```
Comment (GauravGusain98) [inline docs/prerequisites.md:67]:
```suggestion
  - [PHP CS Fixer](https://marketplace.visualstudio.com/items?itemName=junstyle.php-cs-fixer)
```

File: docs/prerequisites.md [hunk 5]
```diff
@@ -54,3 +54,19 @@ You can install it with the following ways:
...
+
+- Sublime Code Editor:
+  - [PHP formatting - PHPfmt](https://packagecontrol.io/packages/phpfmt)
+  - [Laravel Blade Snippets for Balde files](https://marketplace.visualstudio.com/items?itemName=onecentlin.laravel-blade)
```
Comment (GauravGusain98) [inline docs/prerequisites.md:71]:
```suggestion
  - [Laravel Blade Snippets for Blade files](https://marketplace.visualstudio.com/items?itemName=onecentlin.laravel-blade)
```

**Review by GauravGusain98 — CHANGES_REQUESTED** [review]
Please make the requested changes.

**Comment by GauravGusain98** [comment]
@P4NK4J @kuldeep3 Could you please check this PR once.

**Review by P4NK4J — APPROVED** [review]
Looks good.

**Review by GauravGusain98 — APPROVED** [review]
Looks good

**Pull request merged**